clients/fides-js/src/fides-tcf.ts

```diff
@@ -109,30 +111,31 @@ const updateCookie = async (
     return { ...oldCookie, tc_string: "" };
   }
 
-  const tcStringPreferences: TcfSavePreferences = {
-    purpose_preferences: experience.tcf_purposes?.map((purpose) => ({
-      id: purpose.id,
-      preference: getInitialPreference(purpose),
-    })),
-    special_feature_preferences: experience.tcf_special_features?.map(
-      (feature) => ({
-        id: feature.id,
-        preference: getInitialPreference(feature),
-      })
-    ),
-    vendor_preferences: experience.tcf_vendors?.map((vendor) => ({
-      id: vendor.id,
-      preference: getInitialPreference(vendor),
-    })),
-    system_preferences: experience.tcf_systems?.map((system) => ({
-      id: system.id,
-      preference: getInitialPreference(system),
-    })),
-  };
+  // const tcStringPreferences: TcfSavePreferences = {
```

Comment:
looking for a second opinion on the commented out code in this file! the purpose of this logic was for the use case of we have an experience that has user preferences filled in, and now we are populating the cookie/tc string based on those values.

however, with our recent prefetch work, and the decision that TCF needs prefetch (where the cookie determines user preference rather than the backend) I don't think this makes sense anymore. In particular, it is difficult to work with in this PR because the info we get from the backend doesn't have `vendorConsents`/`vendorLegints` yet. 

I'd like to just get rid of it to simplify code paths, but that feels like a big decision!

Reply:
yes you're right! Since TCF requires prefetch, this code path isn't necessary. Let's go ahead and remove this code and leave a code comment describing something like:  for TCF we don't need to update the cookie on init, because we populate the pre-fetched experience with consent from a pre-existing cookie